lib/langchain/llm/ollama.rb

```diff
@@ -27,6 +27,13 @@ def initialize(url:, default_options: {})
       @defaults = DEFAULTS.merge(default_options)
     end
 
+    # Returns the # of vector dimensions for the embeddings
+    # @return [Integer] The # of vector dimensions
+    def default_dimension
+      # since Ollama can run multiple models, generate an embedding and return the size
+      @default_dimension ||= embed(text: "test").embedding.size
```

Comment:
What if instead of dynamically calculating it we'd figure out what this size is for every single supported model and store it in a look up hash? 🤔

Reply:
I'd be willing to build that hash for a popular few... but all? I'm not a fan of that. If I did a partial list, and then fall back to this method, would that be ok?

Reply:
@rickychilcott I'm good with that!